src/Microsoft.FeatureManagement/FeatureFilters/Recurrence/RecurrencePattern.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.FeatureManagement.FeatureFilters
+{
+    /// <summary>
+    /// The recurrence pattern describes the frequency by which the time window repeats.
+    /// </summary>
+    public class RecurrencePattern
+    {
+        /// <summary>
+        /// The recurrence pattern type.
+        /// </summary>
+        public RecurrencePatternType Type { get; set; }
+
+        /// <summary>
+        /// The number of units between occurrences, where units can be in days, weeks, months, or years, depending on the pattern type.
+        /// </summary>
+        public int Interval { get; set; } = 1;
```

Comment:
Why initialize to 1?

Reply:
Yes, the outlook schema doesn't say it has a default value. The reason I set a default value:
1. It will bring more convenience for users who write the JSON configuration directly. Users won't need to write the interval if they just want the interval to be one. I think it would be better to make the job of writing configuration as easy as possible since this is already a very complex schema. And I also guess the interval = 1 is the most common case to set recurrence.
2. The Index property has a default value. It will be more consistent to give Interval a default value.